code/controllers/subsystem/lobotomy_corp.dm

```diff
@@ -121,6 +134,10 @@ SUBSYSTEM_DEF(lobotomy_corp)
 	if(ran_ordeal)
 		return
 	InitiateMeltdown(qliphoth_meltdown_amount, FALSE)
+	if(abno_wait <= qliphoth_state)
+		SSabnormality_queue.fire() // Fired AFTER the meltdown occurs, so there's no situation where players are screwed because they were prepping for CENSORED and he arrives and instant meltdowns.
+		abno_wait_cooldown = round(clamp(1+(player_ratio*player_count), 1, INFINITY)) // Actively dynamic with the living players.
```

Comment:
Why is it scaling up with pop..?

Reply:
When there's a lot of people 2 meltdowns or even 1 can be REALLY LOW and abnos could come in REALLY FAST. I wanna give people a chance to do shit.